packages/issuer/contracts/Issuer.sol

```diff
@@ -60,9 +60,9 @@ contract Issuer is Initializable, Ownable {
 		bytes32 hash;
 	}
 
-    function initialize(int _certificateTopic, address _registry, address _owner) public initializer {
-        require(_registry != address(0), "initialize: Cannot use address 0x0 as registry address.");
-        require(_owner != address(0), "initialize: Cannot use address 0x0 as the owner.");
+    function init(int _certificateTopic, address _registry, address _owner) public initializer {
```

Comment:
Why is the rename applied? This requires us to redeploy etc.

Reply:
For some reason the function with name `initialize` was not available in the typechain autogenerated facade. I renamed it and it worked (dunno why, figured it was a naming conflict with ethers).

But a good point to re-review this. I might try something like this instead: https://github.com/ethers-io/ethers.js/issues/407#issuecomment-458329708.